lib/tasks/import.rake

```diff
@@ -26,6 +26,42 @@ namespace :pqa do
     import_from_mock_server(questions, min_date, max_date)
   end
```

Comment:
Perhaps at this point it would be also handy to have a rake task to import single uins without the need of firing up the Rails console. Perhaps something like

     rake pqa:import_uins[uin1,uin2,...]